metadata-ingestion/tests/unit/sdk_v2/test_datajob.py

```diff
@@ -0,0 +1,117 @@
+import pathlib
+from datetime import datetime, timezone
+
+import pytest
+
+from datahub.emitter.mce_builder import DEFAULT_ENV
+from datahub.metadata.urns import CorpUserUrn, DataJobUrn, TagUrn
+from datahub.sdk.dataflow import DataFlow
+from datahub.sdk.datajob import DataJob
+from datahub.testing.sdk_v2_helpers import assert_entity_golden
+
+_GOLDEN_DIR = pathlib.Path(__file__).parent / "datajob_golden"
+_GOLDEN_DIR.mkdir(exist_ok=True)
+
+
+def test_datajob_basic(pytestconfig: pytest.Config) -> None:
+    # Create a dataflow first
+    flow = DataFlow(
+        platform="airflow",
+        id="example_dag",
+    )
+
+    # Create a basic datajob
+    job = DataJob(
```

Comment:
FYI this shoudl be the simpliest way to create a DataJob (we're getting platform information from flow urn)